src/main/resources/locale_ru.properties

```
@@ -237,7 +237,8 @@ command.roles.info.message.reputationrole=Роли репутации
 command.roles.info.message.roleinfo=Информация о ролях
 command.roles.list.description=Список ролей для репутации.
 command.roles.refresh.description=Обновление ролей пользователей.
-command.roles.refresh.message.finished=Обновление роли пользователя выполнено.
+command.roles.refresh.message.finished=Обновление роли пользователя выполнено.\nVérifié : %CHECKED%\nMis à jour : %UPDATED%\n
```

DeepL suggestion and remove trailing newline:
```suggestion
command.roles.refresh.message.finished=Обновление роли пользователя выполнено.\nПроверено: %CHECKED%\nОбновленный: %UPDATED%
```